modules/dxp/apps/analytics/analytics-reports-document-impl/src/main/java/com/liferay/analytics/reports/document/internal/content/dashboard/item/action/provider/ViewInPanelDocumentContentDashboardItemActionProvider.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.analytics.reports.document.internal.content.dashboard.item.action.provider;
+
+import com.liferay.analytics.reports.info.action.provider.AnalyticsReportsContentDashboardItemActionProvider;
+import com.liferay.content.dashboard.item.action.ContentDashboardItemAction;
+import com.liferay.content.dashboard.item.action.exception.ContentDashboardItemActionException;
+import com.liferay.content.dashboard.item.action.provider.ContentDashboardItemActionProvider;
+import com.liferay.info.item.InfoItemReference;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.repository.model.FileEntry;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Yurena Cabrera
+ */
+@Component(service = ContentDashboardItemActionProvider.class)
+public class ViewInPanelDocumentContentDashboardItemActionProvider
```

Comment:
All the classes should be called `...FileEntry...` instead of `...Document...`. The right replacement for `JournalArticle` is `FileEntry` which is the name of the entity.

Reply:
Done, thanks!